docs/ruleset_functions/normalize_interior_lighting_schedules.md

```diff
@@ -5,6 +5,7 @@ Description: This function would determine a normalized interior lighting schedu
 
 Inputs:
   - **Space.data_element**: The space data element that needs to determine a normalized schedule, e.g. space.interior_lighting.
+  - **adjust_for_credit**: Flag to indicate whether the schedule normalization needs to add control credit adjustment.
```

Comment:
Shouldn't the input be Space.ID and not Space.data_element, since we are looking for a normalized schedule representing all the interior lighting objects associated with a given space?

Reply:
I can update this to space.interior_lightings since the function only focus on lightings, will that work?

Reply:
Update it to space